src/update/provider.cr

```diff
@@ -9,12 +9,30 @@ module Update
 
     macro extended
       PROVIDERS << {{@type}}
+
+      def executable
+        {{@type}}::EXECUTABLE
+      end
+
+      def title
+        {{@type}}::TITLE
```

Comment:
Oh, what about `STUFF`? :)

But we need to write correct some banners.

For example:
```crystal
STUFF = "Atom Packages" # => "Updating Atom Packages"
```

It looks more correct and with fewer tautology.